apis/handlers/restart.go

```diff
@@ -0,0 +1,55 @@
+// Copyright Contributors to the L3AF Project.
+// SPDX-License-Identifier: Apache-2.0
+
+package handlers
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+
+	"net/http"
+
+	"github.com/rs/zerolog/log"
+
+	"github.com/l3af-project/l3afd/v2/bpfprogs"
+	"github.com/l3af-project/l3afd/v2/models"
+)
+
+// HandleRestart Store meta data about ebpf programs and exit
+// @Summary Store meta data about ebpf programs and exit
+// @Description Store meta data about ebpf programs and exit
+// @Accept  json
+// @Produce  json
+// @Param cfgs body []models.L3afBPFPrograms true "BPF programs"
+// @Success 200
+// @Router /l3af/configs/v1/restart [put]
+func HandleRestart(ctx context.Context, bpfcfg *bpfprogs.NFConfigs) http.HandlerFunc {
```

Comment:
Do we use ctx?  If not, please take it away.

Reply:
done